.github/pull_request_template.md

```diff
@@ -34,7 +34,7 @@ If this PR introduces user-facing changes, please note them here.
  - [ ] Add example if relevant.
  - [ ] Document any changes to public APIs.
  - [ ] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
```

Comment:
Also - do we need this step?

Reply:
why not? the labels will make the PR more informative.

Reply:
Maybe we could ask the contributor to categorise their changelog entries based on these tag.

Reply:
Yeah but if we add changelog entries manually - why do we need to do it twice? 
We'll need to add this label in the PR AND add the same in the changelog, right?

Reply:
yes, we should do both, as the label will be useful when we want to search the PR history in the github UI or do categorisation in programmatic way.

While having the category in the changelog makes the changelog more readable.

Reply:
If we edit / maintain changelog manually - we can do `git blame` or use Github UI to find the PR that added each changelog entry as well. 


Another question - how will we remember tags for the manual changelog edits? Should we add the whole list somewhere (e.g. on the top of the file) for the reference?

Reply:
I meant not everyone will have the skills or bother to check the source code/use git blame to check change logs.
As a public repository, adding the label to the PR will also make the PR more descriptive with some metadata that can be easily identified. Which will also allows our customers, PMs or support engineers to use the labels to do some quick search.  I don't see a reason why we don't want to make the PRs as more informative as possible.

> how will we remember tags for the manual changelog edits? Should we add the whole list somewhere (e.g. on the top of the file) for the reference?

Yes, I think we should add some instructions to our documentations, either in DEVELOPING.md, or have a section of description of the categroies in the CHANGELOG.md itself.